frontend/src/language/language.d.ts

```diff
@@ -635,4 +641,70 @@ export type LanguageStrings = {
     noMasterPasswordForDecryption: string;
     loadApiConfigsFailed: string;
   };
+  sns: {
+    characterListTitle: string;
+    accessDenied: string;
+    createPost: string;
+    editPost: string;
+    likes: string;
+    comments: string;
+    secretAccount: string;
+    mainAccount: string;
+    secretAccountWarning: string;
+    selectedSticker: string;
+    changeSticker: string;
+    selectSticker: string;
+    noStickers: string;
+    postPlaceholder: string;
+    posts: string;
+    secrets: string;
+    tags: string;
+    postsTab: string;
+    secretsTab: string;
+    tagsTab: string;
+    addTag: string;
+    tagsDescription: string;
+    postsCount: string;
+    noPostsYet: string;
+    noSecretsYet: string;
+    noTagsYet: string;
```

Comment:
It seems to not match the language keys used in the actual UI.

```suggestion
    noPostsAvailable: string;
    noSecretsAvailable: string;
    noTagsAvailable: string;
```